lib/src/InteractableView.js

```diff
@@ -40,9 +40,9 @@ class WrappedAnimatedInteractableView extends Component {
     }
   }
 
-  componentWillMount() {
-    // this.chokeTheBridge();
-  }
+  // componentWillMount() {
+  //   // this.chokeTheBridge();
+  // }
```

Comment:
delete?

Reply:
I think this should be kept in place for future reference - Updated the comment.